pipeline-templates/job/deploy.yml

```diff
@@ -0,0 +1,51 @@
+parameters:
+  ServiceConnection:
+  SolutionBaseName:
+  Environment:
+  AppRoleAssignmentsServiceConnection:
+
+jobs:
+- deployment: DeployWebApp
+  pool:
+    name: DAS - Continuous Deployment Agents
+  environment: ${{ parameters.Environment }}
+  strategy:
+    runOnce:
+      deploy:
+        steps:
+        - template: azure-pipelines-templates/deploy/step/wait-azure-devops-deployment.yml@das-platform-building-blocks
+          parameters:
+            ServiceConnection: ${{ parameters.ServiceConnection }}
+            EnvironmentId: $(Environment.Id)
+            PipelineName: $(Build.DefinitionName)
+            RunId: $(Build.BuildId)
+            SleepTime: 40        
```

Comment:
Can we use the default time or change the default on the building block?

Reply:
I've used a non default sleeptime because the delivery team wants to deploy to AT on every commit and which will likely have more AT deployments in parallel. The increased sleeptime will avoid any deployment clashes. Changing the Default on the building block could be a consideration but we need to take into account the speed of other pipelines